### PR TITLE
Space quicklook UI on the overview page

### PR DESCRIPTION
**Pull request by msecret**
Finishes overview page by having a quicklook UI for each space in an org, that gets displayed when you click on the org name.

More info in commit messages.

Relies on https://github.com/18F/cg-style/pull/200
Refs https://github.com/18F/cg-dashboard/issues/768

### CODE REVIEW

**Comment by msecret** [comment]
screen shot 2016-12-02 at 4 43 32 pm

File: static_src/components/space_quicklook.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  render() {
+    const space = this.props.space;
+    let loading = <Loading text="Loading spaces" loadingDelayMS={ 1 } />;
+    let content = <div>{ loading }</div>;
+
+    if (!this.props.loading) {
+      content = (
+        <PanelRow>
+          <h3><a href={ this.spaceHref() }>{ space.name }</a></h3>
+          { space.apps && space.apps.map((app) =>
```
Comment (adborden) [inline static_src/components/space_quicklook.jsx]: Are these sorted?

File: static_src/test/unit/stores/space_store.spec.js [hunk 2]
```diff
@@ -73,6 +75,33 @@ describe('SpaceStore', function() {
...
   });
 
+  describe('on fetch all for orgs', function() {
+    it('should call space fetch for each space that belongs to the org', () => {
+      const orgGuid = '240jejbvrl';
+      SpaceStore.push({ guid: 'abc1', organization_guid: orgGuid });
+      SpaceStore.push({ guid: 'abc2', organization_guid: orgGuid });
+      SpaceStore.push({ guid: 'abc3', organization_guid: '23fdg' });
+
+      const spy = sandbox.stub(cfApi, 'fetchSpace').returns(Promise.resolve());
```
Comment (adborden) [inline static_src/test/unit/stores/space_store.spec.js]: `stub` technically ;)

File: static_src/test/unit/stores/space_store.spec.js [hunk 3]
```diff
@@ -73,6 +75,33 @@ describe('SpaceStore', function() {
...
+      const orgGuid = '240jejxcb';
+      SpaceStore.push({ guid: 'abc0', organization_guid: orgGuid });
+
+      const spy = sandbox.spy(SpaceStore, 'emitChange');
+
+      spaceActions.fetchAllForOrg('sdf');
+      spaceActions.fetchAllForOrg(orgGuid);
+
+      expect(spy).toHaveBeenCalledOnce();
+    });
```
Comment (adborden) [inline static_src/test/unit/stores/space_store.spec.js]: I don't see an `emitChange` call in the `SPACES_FOR_ORG_FETCH` handler, is this event from the `load` call? Maybe we should update the test description or add a comment so the test is more clear.

Comment (msecret) [inline static_src/test/unit/stores/space_store.spec.js]: Yea I think it's from the load call, will update.

File: static_src/components/space_quicklook.jsx (hunk 1 above)
Comment (msecret) [inline static_src/components/space_quicklook.jsx]: They are already actually.

**Pull request merged**